graphdatascience/tests/integrationV2/procedure_surface/arrow/conftest.py

```diff
@@ -93,28 +93,42 @@ def arrow_client(session_container: DockerContainer) -> AuthenticatedArrowClient
 
 
 @pytest.fixture(scope="package")
-def neo4j_container(network: Network) -> Generator[DockerContainer, None, None]:
-    neo4j_image = os.getenv("NEO4J_DATABASE_IMAGE")
+def neo4j_container(network: Network, logs_dir: Path, inside_ci: bool) -> Generator[DockerContainer, None, None]:
+    neo4j_image = os.getenv("NEO4J_DATABASE_IMAGE", "neo4j:enterprise")
 
     if neo4j_image is None:
         raise ValueError("NEO4J_DATABASE_IMAGE environment variable is not set")
 
+    db_logs_dir = logs_dir / "arrow_surface" / "db_logs"
+    db_logs_dir.mkdir(parents=True)
+    db_logs_dir.chmod(0o755)
+
     db_container = (
         DockerContainer(image=neo4j_image)
         .with_env("NEO4J_ACCEPT_LICENSE_AGREEMENT", "yes")
         .with_env("NEO4J_AUTH", "neo4j/password")
-        .with_env("NEO4J_server_jvm_additional", "-Dcom.neo4j.arrow.GdsFeatureToggles.enableGds=false")
+        # .with_env("NEO4J_server_jvm_additional", "-Dcom.neo4j.arrow.GdsFeatureToggles.enableGds=false")
```

Comment:
had to comment this out otherwise the built-in remote ops wouldnt be found.
afaik we dont provide a custom remote-ops jar here?